plugins/modules/keycloak_realm_localization.py

```diff
@@ -0,0 +1,391 @@
+# Python
+# !/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# This Ansible module manages realm localization overrides in Keycloak.
+# It ensures the set of message key/value overrides for a given locale
+# either matches the provided list (state=present) or is fully removed (state=absent).
+
+# Copyright (c) 2025, Jakub Danek <danek.ja@gmail.com>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or
+# https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+module: keycloak_realm_localization
+
+short_description: Manage Keycloak realm localization overrides via the Keycloak API
+
+version_added: 10.5.0
+
+description:
+  - Manage per-locale message overrides for a Keycloak realm using the Keycloak Admin REST API.
+  - Requires access via OpenID Connect; the connecting user/client must have sufficient privileges.
+  - The names of module options are snake_cased versions of the names found in the Keycloak API.
+
+attributes:
+  check_mode:
+    support: full
+  diff_mode:
+    support: full
+
+options:
+  locale:
+    description:
+      - Locale code for which the overrides apply (for example, C(en), C(fi), C(de)).
```

Comment:
Please use semantic markup as documented in:
https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#semantic-markup-within-module-documentation

```suggestion
      - Locale code for which the overrides apply (for example, V(en), V(fi), V(de)).
```

Example values for other params also need to be adjusted the same way.